gamelogs-client/src/App.tsx

```diff
@@ -1,35 +1,171 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+interface NewsItem {
+  title: string;
+  date: string;
+  thumbnail: string;
+}
+interface Game {
+  title: string;
+  logo: string;
+  thumbnail: string;
+}
+interface GameCategory {
+  title: string;
+  games: Game[];
+}
```

Comment:
Nice, we might want to put these eventually to a dedicated types directory, or put them in the backend server as an exportable interface, then set that as a dependency here.